.gitlab-ci.yml

```diff
@@ -927,9 +941,8 @@ test_full_integration:
     - tar -xf /tmp/workspace.tar.gz
     - mv /tmp/build_revisions.env /tmp/stage-artifacts .
 
-    # Post job status
+    # for Post job status
```

Comment:
Committed by mistake?

Reply:
Actually I added it by purpose. In `before_script` it's `for Post job status` but the post job status call is moved into `script` section. @lluiscampos does it make sense?

Reply:
I don't understand what you mean, sorry.

The line below (`apk add ...`) is not only for the job status, but general requirements. This comment was for the post call, which now has been moved to `script`. So we can delete it if you want, or leave it (kind of ignoring the problem), but modifying it does not make sense to me.